src/master-playlist-controller.js

```diff
@@ -359,6 +359,11 @@ export class MasterPlaylistController extends videojs.EventTarget {
       this.mainSegmentLoader_.playlist(updatedPlaylist, this.requestOptions_);
       this.updateDuration();
 
+      // kick segmentloader back into gear if the playlist isn't paused
```

Comment:
Though I think it could make sense to provide a bit more context around why we're calling load if it isn't paused (what exactly is kicking it back into gear). Maybe something like `If the player isn't paused, ensure that the segment loader is running, as it is possible that it was temporarily stopped while waiting for a playlist (e.g., in case the playlist errored and we re-requested it).`.